packages/task/src/browser/task-configurations.ts

```diff
@@ -60,7 +61,7 @@ export class TaskConfigurations implements Disposable {
     protected taskCustomizationMap = new Map<string, TaskCustomization[]>();
 
     /** last directory element under which we look for task config */
-    protected readonly TASKFILEPATH = '.theia';
+    protected taskFilePath: string;
```

Comment:
please initialize the field

Reply:
Thanks, it indeed never got initialized. Will definitely fix it.
However the only method where this field is consumed seems to be unused (that's why manual testing didn't spot the mistake).

Reply:
We should remove it and getConfigFileUri, they are both not used and there are better APIs to get these values

Reply:
No problem, will do that